src/client/puppet-service.ts

```diff
@@ -823,6 +820,15 @@ export class PuppetService extends Puppet {
     )(request)
   }
 
+  /**
+   *
+   * Conversation
+   *
+   */
+  public async conversationRead (conversationId: string) {
```

Comment:
Please remove unrelated changes.

If you have to add this method, I don't think it's right to add an empty function body.

You should throw an error to let users know this method has not been implemented.